lib/rules/length-zero-no-unit/README.md

```diff
@@ -1,6 +1,6 @@
 # length-zero-no-unit
 
-Disallow units for zero lengths.
+Disallow units for zero lengths. Ignores `calc` value in favor of `function-calc-no-invalid` rule.
```

Comment:
Minor doc nitpick. Let's revert the change to this sentence and add the following _below_ the example:

```md
This rule ignores lengths within `calc` functions in favor of the [`function-calc-no-invalid`](../function-calc-no-invalid/README.md) rule.
```

This is to be consistent with other rule READMEs e.g. 
https://github.com/stylelint/stylelint/tree/master/lib/rules/font-family-name-quotes#font-family-name-quotes


Otherwise, this PR LGTM.

Reply:
updated